app/main/views.py

```diff
@@ -1,7 +1,33 @@
 from flask import render_template
+from flask import Response
 from . import main
+import json
+from ..models import User
 
 
 @main.route('/')
 def index():
     return render_template('main/index.html')
+
+
+@main.route('/search')
+def search():
+    return render_template('main/search.html')
+
+
+@main.route('/search/<query>')
+def search_query(query):
+    looking_for = "%" + query + "%"
+    users = User.query.filter((User.first_name.ilike(looking_for)) |
+                              User.last_name.ilike(looking_for))\
+        .order_by(User.first_name).all()
+    data = dict()
+    data["results"] = []
```

Comment:
let's use some sexy list comprehension here and say something like
```python
data['results'] = [{ 'title': u.full_name, 'url': '/account/profile/' + u.id } for u in users]
```
(may be a string/int concat problem in there)